README.md

```diff
@@ -4,40 +4,116 @@
+  - **Virtual minibatch**:
+    - The `Trainer` usually does not know if the model is trained with a single example or multiple examples (minibatch), because the examples that are yielded from the dataset are directly forwarded to the model.
+    - When the `virtual_minibatch_size` option is larger than one, the trainer calls the forward and backward step `virtual_minibatch_size` times before applying the gradients. This increases the number of examples over which the gradient is calculated while the memory consumption stays similar, e.g., with `virtual_minibatch_size=2` the gradient is accumulated over two (batched) examples before calling `optimizer.step(); optimizer.zero_grad()`. See [here](doc/virtual_batch_size_multi_gpu.md) for a more thorough explanation.
+- **Logging**: As logging backend, we use [tensorboardX](https://github.com/lanpa/tensorboardX) to generate a `tfevents` file that can be visualized from a [tensorboard](https://github.com/tensorflow/tensorboard). Custom values to be logged can be defined in subclasses of `padertorch.Model`.
+- **Multi-GPU training**: Easily deploy your model onto multiple GPUs to increase the total batch size and speed up the training. See [here](doc/virtual_batch_size_multi_gpu.md#L68) for implementation details and the [example](padertorch/contrib/examples/multi_gpu) for how to enable it.
+- **Support for lazy data preparation**: Ever tired of pre-computing features, taking up time and space on the hard disk? Padertorch works with lazy dataloaders (e.g., [lazy_dataset](https://github.com/fgnt/lazy_dataset)) which extract the features on the fly!
+- **Hands-on examples**: We provide models and training scripts which help you getting started with padertorch.
+
+# Installation
+
+**Requirements**
+- Python 3
+- matplotlib
+- To run the examples:
+  - [sacred](https://github.com/IDSIA/sacred)
+  - torchvision
+  - [pb_bss](http://github.com/fgnt/pb_bss)
+- Installed by padetorch:
+  - torch
+  - tensorboardX
+  - einops
+  - tqdm
+  - natsort
+  - [lazy_dataset](https://github.com/fgnt/lazy_dataset)
+  - IPython
+  - [paderbox](https://github.com/fgnt/paderbox)
+
+```bash
+$ git clone https://github.com/fgnt/padertorch.git
+$ cd padertorch && pip install -e .
+```
+
+# Getting Started
+
+## A Short Explanation of `padertorch.Module` and `padertorch.Model`
+
+You can build your models upon `padertorch.Module` and `padertorch.Model`.
+Both expect a `forward` method which has the same functionality as the `forward` call of  `torch.nn.Module`: It takes data as input, applies some transformations, and returns the network output:
+
+```python
+def forward(self, example):
+    x = example['x']
+    out = transform(x)
+    return out
+```
+
+Additionally, `padertorch.Model` expects a `review` method to be implemented which takes the data and output of the `forward` call as inputs from which it computes the training loss and metrics for logging in tensorboard:
+
+```python
```

Comment:
I think (not sure if this is really the case for everyone) that it is easier to understand if you include the `class MyModel(pt.Model)` part like

```python
class MyModel(pt.Model):
    def forward(self, example):
        x = example['x']
        out = transform(x)
        return out
```
for the method examples.